### PR TITLE
Added some of the 12.20/12.30 update monsters + updated some other monsters

### PR DESCRIPTION
**Pull request by oualid6496**
- Used the 12.20/12.30 monsters as base from Otland updated the items.
- Updated some monsters from the secret library from a repo that was gifted.
- Fixed 2 typo's in items.xml that were conflicting with the updated monsters.

### CODE REVIEW

File: data/monster/quests/feaster_of_souls/irgix_the_flimsy.xml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+		<flag runonhealth="0" />
+		<flag canwalkonenergy="0" />
+	</flags>
+	<attacks>
+		<attack name="melee" interval="2000" min="-100" max="-500" /> 
+	   	<attack name="poison" interval="1500" chance="15" radius="3" target="1" min="-300" max="-500">
+     		<attribute key="shootEffect" value="envenomedarrow"/>
+     	</attack>
+ 		<attack name="energy" interval="1500" chance="25" length="4" spread="0" min="-300" max="-650">
+			<attribute key="areaEffect" value="ENERGY" />
```
Comment (majestyotbr) [inline data/monster/quests/feaster_of_souls/irgix_the_flimsy.xml]: `<attribute key="areaEffect" value="energy" />`

File: data/monster/quests/feaster_of_souls/irgix_the_flimsy.xml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+     		<attribute key="shootEffect" value="envenomedarrow"/>
+     	</attack>
+ 		<attack name="energy" interval="1500" chance="25" length="4" spread="0" min="-300" max="-650">
+			<attribute key="areaEffect" value="ENERGY" /> 
+		</attack>
+		<attack name="death" interval="1500" chance="35"  radius="4" min="-300" max="-650">
+			<attribute key="areaEffect" value="mortarea" />
+		</attack> 
+		<attack name="energy" interval="1500" chance="20"  radius="4" min="-300" max="-650">
+			<attribute key="areaEffect" value="ENERGYAREA" />
```
Comment (majestyotbr) [inline data/monster/quests/feaster_of_souls/irgix_the_flimsy.xml]: `<attribute key="areaEffect" value="energyarea" />`

File: data/monster/quests/feaster_of_souls/unaz_the_mean.xml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+		<flag runonhealth="0" />
+		<flag canwalkonenergy="0" />
+	</flags>
+	<attacks>
+		<attack name="melee" interval="2000" min="-100" max="-500" /> 
+	   	<attack name="poison" interval="1500" chance="15" radius="3" target="1" min="-300" max="-500">
+     		<attribute key="shootEffect" value="envenomedarrow"/>
+     	</attack>
+ 		<attack name="energy" interval="1500" chance="25" length="4" spread="0" min="-300" max="-650">
+			<attribute key="areaEffect" value="ENERGY" />
```
Comment (majestyotbr) [inline data/monster/quests/feaster_of_souls/unaz_the_mean.xml]: `<attribute key="areaEffect" value="energy" />`

File: data/monster/quests/feaster_of_souls/unaz_the_mean.xml [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+     		<attribute key="shootEffect" value="envenomedarrow"/>
+     	</attack>
+ 		<attack name="energy" interval="1500" chance="25" length="4" spread="0" min="-300" max="-650">
+			<attribute key="areaEffect" value="ENERGY" /> 
+		</attack>
+		<attack name="death" interval="1500" chance="35"  radius="4" min="-300" max="-650">
+			<attribute key="areaEffect" value="mortarea" />
+		</attack> 
+		<attack name="energy" interval="1500" chance="20"  radius="4" min="-300" max="-650">
+			<attribute key="areaEffect" value="ENERGYAREA" />
```
Comment (majestyotbr) [inline data/monster/quests/feaster_of_souls/unaz_the_mean.xml]: `<attribute key="areaEffect" value="energyarea" />`

File: data/monster/quests/feaster_of_souls/vok_the_freakish.xml [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+		<flag runonhealth="0" />
+		<flag canwalkonenergy="0" />
+	</flags>
+	<attacks>
+		<attack name="melee" interval="2000" min="-100" max="-500" /> 
+	   	<attack name="poison" interval="1500" chance="15" radius="3" target="1" min="-300" max="-500">
+     		<attribute key="shootEffect" value="envenomedarrow"/>
+     	</attack>
+ 		<attack name="energy" interval="1500" chance="25" length="4" spread="0" min="-300" max="-650">
+			<attribute key="areaEffect" value="ENERGY" />
```
Comment (majestyotbr) [inline data/monster/quests/feaster_of_souls/vok_the_freakish.xml]: `<attribute key="areaEffect" value="energy" />`

File: data/monster/undeads/cursed_prospector.xml [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+		<flag canwalkonfire="1"/>
+		<flag canwalkonpoison="1"/>
+	</flags>
+	<attacks>
+		<attack name="melee" interval="2000" min="-100" max="-200" />
+	   	<attack name="poison" interval="1700" chance="15" radius="3" target="1" min="-300" max="-550">
+     		<attribute key="shootEffect" value="envenomedarrow"/>
+     	</attack>
+ 		<attack name="energy" interval="1700" chance="25" length="4" spread="0" min="-150" max="-550">
+			<attribute key="areaEffect" value="ENERGY" />
```
Comment (majestyotbr) [inline data/monster/undeads/cursed_prospector.xml]: `<attribute key="areaEffect" value="energy" />`

File: data/monster/undeads/cursed_prospector.xml [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+     		<attribute key="shootEffect" value="envenomedarrow"/>
+     	</attack>
+ 		<attack name="energy" interval="1700" chance="25" length="4" spread="0" min="-150" max="-550">
+			<attribute key="areaEffect" value="ENERGY" /> 
+		</attack>
+		<attack name="death" interval="1700" chance="35"  radius="3" min="-150" max="-550">
+			<attribute key="areaEffect" value="mortarea" />
+		</attack>
+		<attack name="holy" interval="1700" chance="35" radius="3" min="-250" max="-550">
+			<attribute key="areaEffect" value="HOLYAREA" />
```
Comment (majestyotbr) [inline data/monster/undeads/cursed_prospector.xml]: `<attribute key="areaEffect" value="holyarea" />`

File: data/monster/undeads/cursed_prospector.xml [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+		</attack>
+		<attack name="energy" interval="2000" chance="13" range="4" radius="4" target="1" min="-400" max="-550">
+			<attribute key="areaEffect" value="ENERGYAREA"/>
```
Comment (MajestyOTBR) [inline data/monster/undeads/cursed_prospector.xml:38]: `<attribute key="areaEffect" value="ènergyarea"/>`

Comment (majestyotbr) [inline data/monster/undeads/cursed_prospector.xml]: `<attribute key="areaEffect" value="energyarea"/>`

File: data/monster/undeads/evil_prospector.xml [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+		<flag canwalkonfire="1"/>
+		<flag canwalkonpoison="1"/>
+	</flags>
+	<attacks>
+		<attack name="melee" interval="2000" min="-100" max="-500" />
+	   	<attack name="poison" interval="1700" chance="15" radius="3" target="1" min="-250" max="-550">
+     		<attribute key="shootEffect" value="envenomedarrow"/>
+     	</attack>
+ 		<attack name="energy" interval="1700" chance="25" length="4" spread="0" min="-250" max="-550">
+			<attribute key="areaEffect" value="ENERGY" />
```
Comment (majestyotbr) [inline data/monster/undeads/evil_prospector.xml]: `<attribute key="areaEffect" value="energy" /> `

File: data/monster/undeads/evil_prospector.xml [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+     		<attribute key="shootEffect" value="envenomedarrow"/>
+     	</attack>
+ 		<attack name="energy" interval="1700" chance="25" length="4" spread="0" min="-250" max="-550">
+			<attribute key="areaEffect" value="ENERGY" /> 
+		</attack>
+		<attack name="death" interval="1700" chance="35"  radius="3" min="-250" max="-550">
+			<attribute key="areaEffect" value="mortarea" />
+		</attack>
+		<attack name="holy" interval="1700" chance="35" radius="3" min="-250" max="-400">
+			<attribute key="areaEffect" value="HOLYAREA" />
```
Comment (majestyotbr) [inline data/monster/undeads/evil_prospector.xml]: `<attribute key="areaEffect" value="holyarea" />`

File: data/monster/undeads/evil_prospector.xml [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+		</attack>
+		<attack name="energy" interval="1700" chance="13" range="4" radius="4" target="1" min="-400" max="-450">
+			<attribute key="areaEffect" value="ENERGYAREA"/>
```
Comment (majestyotbr) [inline data/monster/undeads/evil_prospector.xml]: `<attribute key="areaEffect" value="energyarea"/>`

File: data/monster/undeads/flimsy_lost_soul.xml [hunk 12]
```diff
@@ -0,0 +1,65 @@
...
+		<flag canwalkonfire="1"/>
+		<flag canwalkonpoison="1"/>
+	</flags>
+	<attacks>
+		<attack name="melee" interval="2000" min="-100" max="-500" /> 
+	   	<attack name="poison" interval="1700" chance="15" radius="3" target="1" min="-150" max="-550">
+     		<attribute key="shootEffect" value="envenomedarrow"/>
+     	</attack>
+ 		<attack name="energy" interval="1700" chance="25" length="4" spread="0" min="-150" max="-550">
+			<attribute key="areaEffect" value="ENERGY" />
```
Comment (majestyotbr) [inline data/monster/undeads/flimsy_lost_soul.xml]: `<attribute key="areaEffect" value="energy" /> `

File: data/monster/undeads/freakish_lost_soul.xml [hunk 13]
```diff
@@ -0,0 +1,65 @@
...
+		<flag canwalkonfire="1"/>
+		<flag canwalkonpoison="1"/>
+	</flags>
+	<attacks>
+		<attack name="melee" interval="2000" min="-100" max="-500" /> 
+	   	<attack name="poison" interval="1700" chance="15" radius="3" target="1" min="-300" max="-550">
+     		<attribute key="shootEffect" value="envenomedarrow"/>
+     	</attack>
+ 		<attack name="energy" interval="1700" chance="25" length="4" spread="0" min="-150" max="-550">
+			<attribute key="areaEffect" value="ENERGY" />
```
Comment (majestyotbr) [inline data/monster/undeads/freakish_lost_soul.xml]: `<attribute key="areaEffect" value="energy" /> `

File: data/monster/undeads/freakish_lost_soul.xml [hunk 14]
```diff
@@ -0,0 +1,65 @@
...
+     		<attribute key="shootEffect" value="envenomedarrow"/>
+     	</attack>
+ 		<attack name="energy" interval="1700" chance="25" length="4" spread="0" min="-150" max="-550">
+			<attribute key="areaEffect" value="ENERGY" /> 
+		</attack>
+		<attack name="death" interval="1700" chance="35"  radius="4" min="-150" max="-550">
+			<attribute key="areaEffect" value="mortarea" />
+		</attack> 
+		<attack name="energy" interval="1700" chance="20"  radius="4" min="-150" max="-550">
+			<attribute key="areaEffect" value="ENERGYAREA" />
```
Comment (majestyotbr) [inline data/monster/undeads/freakish_lost_soul.xml]: `<attribute key="areaEffect" value="energyarea" />`

File: data/monster/undeads/mean_lost_soul.xml [hunk 15]
```diff
@@ -0,0 +1,63 @@
...
+		<flag canwalkonfire="1"/>
+		<flag canwalkonpoison="1"/>
+	</flags>
+	<attacks>
+		<attack name="melee" interval="2000" min="-100" max="-500" /> 
+	   	<attack name="poison" interval="1700" chance="15" radius="3" target="1" min="-300" max="-550">
+     		<attribute key="shootEffect" value="envenomedarrow"/>
+     	</attack>
+ 		<attack name="energy" interval="1700" chance="25" length="4" spread="0" min="-300" max="-550">
+			<attribute key="areaEffect" value="ENERGY" />
```
Comment (majestyotbr) [inline data/monster/undeads/mean_lost_soul.xml]: `<attribute key="areaEffect" value="energy" />`

File: data/monster/quests/feaster_of_souls/irgix_the_flimsy.xml (hunk 1 above)
Comment (oualid6496) [reply data/monster/quests/feaster_of_souls/irgix_the_flimsy.xml]: ty, fixed

File: data/monster/quests/feaster_of_souls/irgix_the_flimsy.xml (hunk 2 above)
Comment (oualid6496) [reply data/monster/quests/feaster_of_souls/irgix_the_flimsy.xml]: ty, fixed

File: data/monster/quests/feaster_of_souls/vok_the_freakish.xml (hunk 5 above)
Comment (oualid6496) [reply data/monster/quests/feaster_of_souls/vok_the_freakish.xml]: ty, fixed

File: data/monster/undeads/cursed_prospector.xml (hunk 7 above)
Comment (oualid6496) [reply data/monster/undeads/cursed_prospector.xml]: ty, fixed

File: data/monster/undeads/evil_prospector.xml (hunk 9 above)
Comment (oualid6496) [reply data/monster/undeads/evil_prospector.xml]: ty, fixed

File: data/monster/undeads/mean_lost_soul.xml (hunk 15 above)
Comment (oualid6496) [reply data/monster/undeads/mean_lost_soul.xml]: ty, fixed

File: data/monster/undeads/freakish_lost_soul.xml (hunk 14 above)
Comment (oualid6496) [reply data/monster/undeads/freakish_lost_soul.xml]: ty, fixed

File: data/monster/undeads/freakish_lost_soul.xml (hunk 13 above)
Comment (oualid6496) [reply data/monster/undeads/freakish_lost_soul.xml]: ty, fixed

File: data/monster/undeads/flimsy_lost_soul.xml (hunk 12 above)
Comment (oualid6496) [reply data/monster/undeads/flimsy_lost_soul.xml]: ty, fixed

File: data/monster/undeads/evil_prospector.xml (hunk 11 above)
Comment (oualid6496) [reply data/monster/undeads/evil_prospector.xml]: ty, fixed

File: data/monster/undeads/evil_prospector.xml (hunk 10 above)
Comment (oualid6496) [reply data/monster/undeads/evil_prospector.xml]: ty, fixed

File: data/monster/undeads/cursed_prospector.xml (hunk 8 above)
Comment (oualid6496) [reply data/monster/undeads/cursed_prospector.xml]: ty, fixed

File: data/monster/undeads/cursed_prospector.xml (hunk 6 above)
Comment (oualid6496) [reply data/monster/undeads/cursed_prospector.xml]: ty, fixed

File: data/monster/quests/feaster_of_souls/unaz_the_mean.xml (hunk 4 above)
Comment (oualid6496) [reply data/monster/quests/feaster_of_souls/unaz_the_mean.xml]: ty, fixed

File: data/monster/quests/feaster_of_souls/irgix_the_flimsy.xml (hunk 2 above)
Comment (oualid6496) [reply data/monster/quests/feaster_of_souls/irgix_the_flimsy.xml]: ty, fixed

**Review by majestyotbr — APPROVED** [review]

**Pull request merged**